assets/js/base/components/drawer/style.scss

```diff
@@ -109,44 +109,48 @@ $drawer-animation-duration: 0.3s;
 	}
 }
 
-.wc-block-components-drawer .components-modal__content {
-	padding: $gap-largest $gap;
+// Important rules are needed to reset button styles.
+.wc-block-components-drawer__close {
+	@include reset-box();
+	background: transparent !important;
+	color: inherit !important;
+	position: absolute !important;
+	top: $gap-small;
+	right: $gap-small;
+	opacity: 0.6;
+	z-index: 2;
+	// Increase clickable area.
+	padding: 1em !important;
+	margin: -1em;
+
+	&:hover,
+	&:focus,
+	&:active {
+		opacity: 1;
+	}
+
+	> span {
+		@include visually-hidden();
+	}
+	svg {
+		fill: currentColor;
+		display: block;
+	}
 }
 
-.wc-block-components-drawer .components-modal__header {
+.wc-block-components-drawer__content {
+	height: 100dvh;
 	position: relative;
+}
 
-	// Close button.
-	.components-button {
-		@include reset-box();
-		background: transparent;
-		color: inherit;
-		position: absolute;
-		opacity: 0.6;
-		z-index: 2;
-		// The SVG has some white spacing around, thus we have to set this magic number.
-		right: 8px;
-		top: 0;
-		// Increase clickable area.
-		padding: 1em;
-		margin: -1em;
-
-		&:hover,
-		&:focus,
-		&:active {
-			opacity: 1;
-		}
-
-		> span {
-			@include visually-hidden();
-		}
-	}
+.admin-bar .wc-block-components-drawer__content {
```

Comment:
This has been moved from here:

https://github.com/woocommerce/woocommerce-blocks/blob/f1714a70bb68d7bae57e7e8a460684c6c5f031ed/assets/js/blocks/mini-cart/style.scss#L228-L238